### PR TITLE
Converions of HTML to Markdown files

### PR DESCRIPTION
**Pull request by delisma**
## Preview link
https://deploy-preview-173--systeme-conception-canada-ca.netlify.app/test

English equivalent is coming

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,91 +1,114 @@
...
       breadcrumbs:
-      - title: À propos de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos.html
-      - title: Système de conception de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
-      - title: Bibliothèque de modèles et de configurations de conception
-        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
+        - title: À propos de Canada.ca
+          link: https://www.canada.ca/fr/gouvernement/a-propos.html
+        - title: Système de conception de Canada.ca
```
Comment (prycrane) [inline _config.yml:85]: "Système de conception de Canada.ca"  should be shortened to "Système de conception":

See: https://test.canada.ca/experimental/prycrane/breadcrumbs/breadcrumbs-fr.html

File: _config.yml [hunk 2]
```diff
@@ -1,91 +1,114 @@
...
+          link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
+        - title: Bibliothèque de modèles et de configurations de conception
```
Comment (prycrane) [inline _config.yml:87]: "Bibliothèque de modèles et de configurations de conception"  should be shortened to "Modèles et conception":

See: https://test.canada.ca/experimental/prycrane/breadcrumbs/breadcrumbs-fr.html

File: _config.yml (hunk 1 above)
Comment (delisma) [reply _config.yml:85]: Should I shorten 'À propos de Canada.ca' as well?

**Comment by donohuech** [comment]
On the main page, the filter bar is appearing in English instead of French. 

https://deploy-preview-173--systeme-conception-canada-ca.netlify.app/test

It should say: Filtrer les articles ..... Affiche 1 à 25 de 157 entrées ....... Afficher 25 entrées

But instead it says: Filter items .... Showing 1 to 25 of 157 entries ...... Show 25 entries

**Comment by donohuech** [comment]
The page feedback tool and the 'partagez cette page' appears on the live site but not on the netlify preview: 

https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html 
https://deploy-preview-173--systeme-conception-canada-ca.netlify.app/test

**Comment by donohuech** [comment]
This page has an exposed </img> right below the actual image: https://deploy-preview-173--systeme-conception-canada-ca.netlify.app/amelioration-continue/recherche.html

**Comment by delisma** [comment]
Closing this request to avoid rebasing and conflicting files. Will link from the new PR.

**Pull request closed without merging**